### ISSUE TITLE
Response Emails Saving In RD

### ISSUE THREAD

**Issue by BG-RLTech** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#845]
We have found with the 2 Way Messaging customers responses to messages automatically save in RD in a tree like conversation. This feature is amazing and shows continuity of conversation. 

Can this same process be applied to the email functions. If we send a email and the customer replies it is saved in the location the email was sent from (Customer Profile, Ticket, Lead). 

There is currently no proof that a customer acknowledged something by email unless we manually transfer the data into RD from the Outlook Email Client.

**Comment by Uzair-RepairDesk** [comment]
Hi @benjamingartner 

Thank you for your appreciation regarding 2 way SMS functionality and sharing your idea regarding email. 
We are already aware of the above request however 2-way email needs our own "Email Server" that required extra resources and involved cost factor. 2 way SMS is possible because we have integrated RepairDesk with 3rd party service providers. Based on our current pricing model, currently its not feasible for us to develop the email server.

However, I have marked this request in enhancements so we can track it in the future. Thanks
